library/src/com/google/maps/android/kml/KmlLayer.java

```diff
@@ -24,26 +26,46 @@
      * @param map        GoogleMap object
      * @param resourceId Raw resource KML file
      * @param context    Context object
+     * @param directoryName the fully qualified directory name to look in (in the android file
+     *                      system) for any relative-path images.
      * @throws XmlPullParserException if file cannot be parsed
      */
-    public KmlLayer(GoogleMap map, int resourceId, Context context)
+    public KmlLayer(GoogleMap map, int resourceId, Context context, String directoryName)
```

Comment:
We should avoid breaking the current API functionality. Can we overload this constructor to work as it did before as well as this version with the `directoryName` parameter?